src/cdk-experimental/column-resize/resize-strategy.ts

```diff
@@ -175,6 +219,26 @@ export class CdkFlexTableResizeStrategy extends ResizeStrategy implements OnDest
   }
 }
 
+/** Converts CSS pixel values to numbers, eg "123px" to 123. */
+function coercePixelsFromCssValue(cssValue: string): number {
+  return Number(cssValue.match(/(\d+)px/)?.[1]);
```

Comment:
Couldn't we get a `NaN` here if it's invalid?

Reply:
Updated the jsdoc to reflect that